build/grunt.js

```diff
@@ -200,14 +166,6 @@ module.exports = function(grunt) {
         dest: 'dist/video-js-' + version.full + '.zip'
       }
     },
-    version: {
```

Comment:
i'm not sure that this actually did anything, I see no difference in master with it removed